src/server/server.js

```diff
@@ -24,7 +24,7 @@ server.use(
 );
 
 server.use("/authentication", authenticationRoute);
-
+server.use("/posts", postsRoute);
```

Comment:
Tror jag har lagt in den i min senaste pull request också, vet inte om det kan bli en konflikt senare men den behövs iaf 😊